pkg/prediction/config/broadcast.go

```diff
@@ -1,37 +1,41 @@
 package config
 
-type broadcast struct {
-	c chan broadcast
-	v interface{}
+type event struct {
```

Comment:
Would you please add some unit test for it?

Reply:
broadcast_test.go